### DISCUSSION TITLE
Unable to bulk import from interface

### DISCUSSION THREAD

**Discussion by tzrafi** [netbox-community/netbox#17205] [Help Wanted!]
I want to import through CSV, but to know the format I export first and then fill each field as per our requirement. But facing some error here, please help.

![image](https://github.com/user-attachments/assets/5fbc6e09-9bf8-42b5-8e77-789f3a672bf0)

![image](https://github.com/user-attachments/assets/bda5260e-8207-41de-b8fb-223c981adb11)

**Comment by candlerb** [comment]
1. Netbox CSV import is case-sensitive. As the error messages say, you need columns called "device", "name", "type" etc, not "Device", "Name", "Type".
2. Netbox CSV import is completely incompatible with CSV export. They [intentionally](https://github.com/netbox-community/netbox/wiki/Frequently-Asked-Questions#why-cant-i-pass-exported-data-directly-into-the-import-form) use different column names and different data formats. Therefore, you cannot use a CSV export as a guide to CSV import. You wouldn't be the first person to be surprised by this design decision.

**Reply by jeremystretch** [reply]
> You wouldn't be the first person to be surprised by this design decision.

Hence the [FAQ](https://github.com/netbox-community/netbox/wiki/Frequently-Asked-Questions#why-cant-i-pass-exported-data-directly-into-the-import-form).
